app/src/main/java/com/github/se/gatherspot/ui/eventUI/EventView.kt

```diff
@@ -124,221 +123,191 @@ fun EventUI(
+                        painter = painterResource(id = R.drawable.delete),
+                        contentDescription = "Delete event")
                   }
-                }
+              ExportToCalendarIcon(event)
+            })
+      }) { innerPadding ->
+        Column(
+            modifier =
+                Modifier.padding(innerPadding)
+                    .padding(8.dp)
+                    .testTag("eventColumn")
+                    .verticalScroll(rememberScrollState())) {
+              EventBody(
+                  event = event,
+                  organizerProfile = organizerProfile,
+                  organizerRating = organizerRating,
+                  navActions = navActions)
+              EventRating(eventRating = eventRating)
+            }
+      }
+  if (showDialogDelete!!) {
+    AlertDialog(
+        modifier = Modifier.testTag("alertBox"),
+        onDismissRequest = { eventUIViewModel.dismissAlert() },
+        title = { Text("Delete Event") },
+        text = {
+          Text("Are you sure you want to delete this event? This action cannot be undone.")
+        },
+        confirmButton = {
+          Button(
+              modifier = Modifier.testTag("okButton"),
+              onClick = {
+                // Delete the event
+                eventUtils.deleteEvent(event)
+                navActions.goBack()
+                eventUIViewModel.dismissAlert()
+              }) {
+                Text("Delete")
               }
-
-              // Inscription Limit Date and Time
-              Spacer(modifier = Modifier.height(8.dp))
-              Text(
-                  "Inscription Limit:",
-                  fontWeight = FontWeight.Bold,
-                  modifier = Modifier.testTag("inscriptionLimitTitle"))
-              Row(modifier = Modifier.testTag("inscriptionLimitDateAndTime")) {
-                event.inscriptionLimitDate?.let { limitDate ->
-                  Text(limitDate.format(DateTimeFormatter.ofLocalizedDate(FormatStyle.MEDIUM)))
-                }
-                Spacer(modifier = Modifier.width(8.dp))
-                event.inscriptionLimitTime?.let { limitTime ->
-                  Text(limitTime.format(DateTimeFormatter.ofLocalizedTime(FormatStyle.SHORT)))
-                }
+        },
+        dismissButton = {
+          Button(
+              modifier = Modifier.testTag("cancelButton"),
+              onClick = { eventUIViewModel.dismissAlert() }) {
+                Text("Cancel")
               }
-              EventQRCodeUI(event = event)
+        })
+  }
+}
 
-              // Rating
-              if (eventUIViewModel.canRate()) {
-                RatingDisplay(
-                    ownRating = ownRating ?: Rating.UNRATED,
-                    eventUIViewModel = eventUIViewModel,
-                    eventRating = eventRating ?: 0.0)
-              }
+@Composable
+fun EventRating(eventRating: Double?) {
+  if ((eventRating ?: 0.0) > 0.0) {
```

Comment:
Yes we don't show anything because we don't enter the condition if the eventRating is 0.0. 
I just use 0.0 as a "don't show" becasue
1. it is more convenient than null that can possibly cause exceptions somewhere
2. With our grading scheme (1-5) the grade 0.0 is unattainable any other way

In theory null would be the more semantically correct representation of this case